Content.Server/Cluwne/CluwneComponent.cs

```diff
@@ -0,0 +1,50 @@
+using Content.Server.Speech.Components;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Cluwne;
+
+[RegisterComponent]
+public sealed class CluwneComponent : Component
```

Comment:
Components should be shared unless you have good reason not to.

Reply:
Does this mean server.cluwnecomponent should actually be a shared.cluwnecomponent?

Reply:
yes

Reply:
cluwne component is now shared